.project

```diff
@@ -23,7 +23,7 @@
 	</linkedResources>
 	<filteredResources>
 		<filter>
-			<id>1555091862383</id>
+			<id>1576009800852</id>
```

Comment:
Just curious, where does this number come from?

Reply:
IDK... looks like a timestamp. These files are managed by eclipse. I.e. I made the change in the UI and it updated the files.